src/test/java/org/nuxeo/segment/io/test/TestSegmentIOService.java

```diff
@@ -46,108 +47,123 @@
     EventService eventService;
 
     protected void sendAuthenticationEvent(Principal principal) throws Exception {
-            Map<String, Serializable> props = new HashMap<String, Serializable>();
-            props.put("AuthenticationPlugin", "FakeAuth");
-            props.put("LoginPlugin", "FakeLogin");
-            EventContext ctx = new UnboundEventContext(principal, props);
-            eventProducer.fireEvent(ctx.newEvent("loginSuccess"));
+        Map<String, Serializable> props = new HashMap<String, Serializable>();
+        props.put("AuthenticationPlugin", "FakeAuth");
+        props.put("LoginPlugin", "FakeLogin");
+        EventContext ctx = new UnboundEventContext(principal, props);
+        eventProducer.fireEvent(ctx.newEvent("loginSuccess"));
     }
 
-
     @Test
     public void checkDeploy() {
-
-        SegmentIOComponent component = (SegmentIOComponent) Framework.getRuntime().getComponent(SegmentIOComponent.class.getName());
-        Assert.assertNotNull(component);
-
-        SegmentIO service = Framework.getLocalService(SegmentIO.class);
+        SegmentIO service = Framework.getService(SegmentIO.class);
         Assert.assertNotNull(service);
 
-        Map<String, List<SegmentIOMapper>> allMappers = component.getAllMappers();
-        Assert.assertTrue(allMappers.size()>0);
+        Map<String, List<SegmentIOMapper>> allMappers = service.getAllMappers();
+        Assert.assertTrue(allMappers.size() > 0);
         Assert.assertTrue(allMappers.containsKey("loginSuccess"));
-        Assert.assertTrue(allMappers.get("loginSuccess").size()>0);
+        Assert.assertTrue(allMappers.get("loginSuccess").size() > 0);
         Assert.assertTrue(allMappers.get("loginSuccess").get(0).getName().equals("testIdentify"));
 
     }
 
     @Test
-    public void shouldRunIdentifyOnLogin() throws Exception {
+    public void shouldRunIdentifyAndGroupOnLogin() throws Exception {
 
-        SegmentIOComponent component = (SegmentIOComponent) Framework.getRuntime().getComponent(SegmentIOComponent.class.getName());
+        SegmentIOComponent component = (SegmentIOComponent) Framework.getService(SegmentIO.class);
```

Comment:
I'm wondering if we could use `Framework.getService(SegmentIOComponent.class)` directly?